docs/ReleaseNotes.md

```diff
@@ -1,6 +1,22 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.3.2
+Acuminator 2.3.2 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 2.3.2, the following enhancements have been implemented:
+ - The [PX1076](diagnostics/PX1076.md) diagnostic has been added to check for and warn a developer that the PXInternalUseOnly attribute is used in the customization code.
```

Comment:
This diagnostic does not check the `PXInternalUseOnly` attribute usage. 

Some Acumatica APIs are effectively internal but can't have `internal` access modifier because they are accessed from different projects. Such APIs has to be `public` from the C# point of view. This means that partner developers can access them easily without resorting to complex means such as .Net Reflection. However, we don't want external developers to use these internal APIs. We mark such APIs with `PXInternalUseOnly` attribute to emphasize that they are internal to Acumatica and should not be used in external solutions - customizations, integrations, etc. We won't maintain such internal APIs - they can be changed or deleted in any minor upgrade.  
The downside of the attribute approach is that the developer needs to check the declaration of any used Acumatica API for the presence of `PXInternalUseOnly`attribute. This is tiresome and is frequently forgotten leading to a situation when some external solutions rely on internal Acumatica APIs. The PX1076 diagnostic addresses this flaw by displaying a warning to external developers for all used Acumatica APIs marked with `PXInternalUseOnly`attribute. 

Note that it has nothing to do with the usage of `PXInternalUseOnly`attribute itself in the code of customizations or other external solutions. It is not recommended to mark new code in external solutions with this attribute but it is ok to check in the customization code if this attribute is defined on some class or property